build/media-suite_helper.sh

```diff
@@ -1909,6 +1911,14 @@ EOF
         printf '%s\r\n' "@echo off" "" "bash $LOCALDESTDIR/bin/ab-pkg-config --static %*" > "$LOCALDESTDIR"/bin/ab-pkg-config-static.bat
 }
 
+create_ab_ccache() {
+    local bin
+    for bin in gcc g++ cc cpp c++; do
+        [[ -f "$LOCALDESTDIR"/bin/"$bin".bat ]] ||
+           printf '%s\r\n' "@echo off" "" "bash $MINGW_PREFIX/lib/ccache/bin/$bin %*" > "$LOCALDESTDIR"/bin/"$bin".bat
```

Comment:
Why don't we just call ccache directly with the $bin as the second argument?

Reply:
waf doesn't work with 'ccache gcc', at least.